ui/elastic/src/App.js

```diff
@@ -0,0 +1,260 @@
+import React, { Component } from 'react';
+import './App.css';
+
+import {
+    SearchBox,
+    NoHits,
+    Hits,
+    HitsStats,
+    ItemCheckboxList,
+    SortingSelector,
+    SelectedFilters,
+    MenuFilter,
+    Pagination,
+    RefinementListFilter,
+    ResetFilters,
+    SearchkitManager,
+    SearchkitProvider,
+    Tabs,
+    } from "searchkit";
+
+import {
+  Layout, TopBar, LayoutBody, LayoutResults,
+  ActionBar, ActionBarRow, SideBar
+} from "searchkit"
+
+const searchkit = new SearchkitManager("https://kinto-ota.dev.mozaws.net/v1/buckets/build-hub/collections/releasesv2/", {searchUrlPath: "search"})
+
+
+const HitsTable = (toggleExpand, expandedEntry) => {
+  const tableStyle = {width: "100%", boxSizing: 'border-box'};
+  return ({hits}) => {
+    return (
+      <div style={{width: '100%', boxSizing: 'border-box', padding: 8}}>
+        <table className="sk-table sk-table-striped" style={tableStyle}>
+          <thead>
+            <tr>
+              <th>Product</th>
+              <th>Version</th>
+              <th>Platform</th>
+              <th>Channel</th>
+              <th>Locale</th>
+              <th>Published on</th>
+            </tr>
+          </thead>
+          <tbody>
+            {hits.map(hit => {
```

Comment:
Nit: you could probably destructure `source`, `target` and friends from `_source` to reduce verbosity:

```js
{hits.map(({_source: {build, download, source, target}}) => {
```